packages/settings-library/src/settings_library/utils_encoders.py

```diff
@@ -0,0 +1,15 @@
+from functools import partial
+
+from pydantic import BaseModel, SecretStr
+from pydantic.json import custom_pydantic_encoder
+
+
+def create_json_encoder_wo_secrets(model_cls: type[BaseModel]):
```

Comment:
Perhaps you can add a small doc indicating that this function can be handy to export instances of `BaseModel`, that have embedded secrets, 
```python
model.dict()['my_secret'] == "********"

show_secrets_encoder = create_json_encoder_wo_secrets(type(model))
model.dict(encoder=show_secrets_encoder)['my_secret'] == "secret"
```